plugins/services/services_inspector.rb

```diff
@@ -87,6 +93,22 @@ def inspect_sysvinit_services
     ServiceList.new(services.sort_by(&:name))
   end
 
+  def inspect_upstart_services
+    output = @system.run_command("/sbin/initctl", "show-config", "-e", :stdout => :capture)
+
+    servicelist = output.lines.map(&:chomp).slice_before { |l| !l.start_with?(" ") }
+    enabled,disabled = servicelist.partition { |s| s.find { |e| e.start_with?("  start on runlevel", "  start on startup") }}
```

Comment:
Line is too long. [125/100]<br>Space missing after comma.<br>Space missing inside }.